pwndbg/gdblib/arch.py

```diff
@@ -44,16 +44,18 @@
 
 def _get_arch(ptrsize: int):
     not_exactly_arch = False
+    endian_value = gdb.execute("show endian", to_string=True)
 
-    if "little" in gdb.execute("show endian", to_string=True).lower():
+    if endian_value and "little" in endian_value.lower():
```

Comment:
Any reason why we remove `.lower()` here? do we know if it can't be upper?

Reply:
We did not remove `.lower()`